src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -49,86 +60,112 @@ def llm_eval_binary(
     return [_snap_to_rail(response, rails_set) for response in responses]
 
 
+"""
+
+Args:
+    dataframe (pd.DataFrame):
+
+    rails_map: The mapping to use to return the values. Using a dict here
+    but it can take the OrderedDict of RAILS_MAP from library
+
+Returns:
+    List[List[str]]:
+"""
```

Comment:
floating doc string? Not sure where this is supposed to go?

Reply:
Whoops, forgot to remove